hypha/templates/includes/user_menu.html

```diff
@@ -1,67 +1,75 @@
 {% load i18n heroicons %}
 
 {% if request.user.is_authenticated %}
-    <div x-data="{ show: false }" class="relative flex">
+    <div x-data="{ show: false }" x-on:keydown.escape="show = false" class="relative flex">
         <button
             x-on:click="show = ! show"
-            class="font-bold flex items-center ps-1 pe-4 hover:bg-black hover:text-white transition-colors rounded-sm"
+            class="font-bold flex items-center transition-all rounded-sm px-1.5 bg-gray-100"
             type="button"
+            :class="show ? 'bg-gray-900 text-white' : 'hover:bg-light-blue/30'"
```

Comment:
```suggestion
            :class="show ? 'bg-gray-900 text-white' : 'hover:bg-slate-200'"
```

To match the notification bell hover state